### ISSUE TITLE
Any-to-red/yellow/green/blue translation tables don't work in BTSX

### ISSUE THREAD

**Issue by dististik** [fabiangreffrath/woof#933]
I'm not sure if this is a combination of BTSX's altered playpal and the STTNUM* lumps being silver, but for both episodes the new translation tables don't alter the color of those graphics or STCFN*** graphics in episode 2. The tables do work for pure white graphics like Eviternity's STTNUM* lumps, though.

**Comment by fabiangreffrath** [comment]
Yes, this is because BTSX already provides its own color translation tables and the port never overrides PWAD author's choice. 😇 However, in BTSX's own color translation tables there is *no* color translation prepared from the gray range to any other color. This means that it's the PWAD author's explicit choice that status bar numbers are not subject to color translation.

TL;DR use the `-tranmap` parameter to force recreation of the color translation tables. :wink:

**Comment by dististik** [comment]
Ah right, my bad! Thanks for the reply; I'll try it out with the parameter 🙂
